File: abc/abc338/a.py
# https://atcoder.jp/contests/abc338/submissions/49692359

s = input()

if not('A' <= s[0] <= 'Z'):
  print('No')
  exit()

for i in range(1, len(s)):
  if not('a' <= s[i] <= 'z'):
    print('No')
    exit()

print('Yes')

File: abc/abc338/a.cs
// https://atcoder.jp/contests/abc338/submissions/49753829

using static System.Console;
using System.Text.RegularExpressions;

public class A
{
  static void Main()
  {
    string s = ReadLine();
    Write(Regex.IsMatch(s, "^[A-Z][a-z]*$") ? "Yes" : "No");
  }
}
